FindmallocMC.cmake

```diff
@@ -97,44 +102,47 @@ if(mallocMC_ROOT_DIR)
     # mallocMC variables ########################################################
     #
     set(mallocMC_VERSION "${mallocMC_VERSION_MAJOR}.${mallocMC_VERSION_MINOR}.${mallocMC_VERSION_PATCH}")
-    set(mallocMC_INCLUDE_DIRS ${mallocMC_ROOT_DIR}/include)
+    set(mallocMC_INCLUDE_DIRS ${mallocMC_ROOT_DIR}/include CACHE PATH "mallocMC include pathes")
```

Comment:
probably we can keep them non-cached, there is no direct use for the user to "modify" version and libraries/includes by hand (well, a bit for the latter) but since they are all printed that is good enough.